CHANGELOG.md

```diff
@@ -14,7 +14,7 @@
 - [Reverted] Use macos version 11.0 to run instrumentation tests on CI
 - Migrate `RecentPatientsScreen` to a fragment
 - Paginate recent patients list
-- Pin Android emulator build version to resolve emulator timeout error
+- [Reverted] Pin Android emulator build version to resolve emulator timeout error
```

Comment:
If we reverted it in this release itself, then we could just delete the line instead of marking it as `[Reverted]`. What do you think?

Reply:
Same comment for the older line related to the macos version.

Reply:
Yeah, we can do that. I was just thinking of keeping track of something changes we made and reverted, that's all.